cloud-sql/mysql/sqlalchemy/deployment.yaml

```
@@ -0,0 +1,66 @@
+# [START cloud_sql_mysql_sqlalchemy_gke_quickstart_deployment]
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gke-cloud-sql-quickstart
+spec:
+  selector:
+    matchLabels:
+      app: gke-cloud-sql-app
+  template:
+    metadata:
+      labels:
+        app: gke-cloud-sql-app
+    spec:
+      serviceAccountName: <YOUR-KSA-NAME>
+      containers:
+      - name: gke-cloud-sql-app
+        # Replace $LOCATION with your Artifact Registry location (e.g., us-central1).
+        # Replace $YOUR_PROJECT_ID with your project ID.
+        image: $LOCATION-docker.pkg.dev/$YOUR_PROJECT_ID/gke-cloud-sql-repo/gke-sql:latest
+        # This app listens on port 8080 for web traffic by default.
+        ports:
+        - containerPort: 8080
+        env:
+        - name: PORT
+          value: "8080"
+        - name: DB_HOST
+          value: "0.0.0.0"

```

Should this be localhost? or a filler value? 
```suggestion
          value: "0.0.0.0"

```

---

Updated to be 
value: "127.0.0.1"